airflow/contrib/hooks/gcs_hook.py

```diff
@@ -187,8 +189,15 @@ def upload(self, bucket, object, filename,
         :type mime_type: str
         :param gzip: Option to compress file for upload
         :type gzip: bool
+        :param multipart: Deprecated parameter. Multipart would be handled automatically
+        :type multipart: bool or int
+        :param num_retries: Deprecated parameter. Retries would be handled automatically
+        :type num_retries: int
         """
 
+        warnings.warn("'multipart' and 'num_retries' parameters have been deprecated."
```

Comment:
I would suggest passing `multipart=None` and then an `if multipart is not None` -- otherwise there is no way for the user to "fix" this warning.

Reply:
Done. Updated